Cargo.toml

```diff
@@ -186,9 +192,8 @@ cid = { version = "0.10.1", default-features = false, features = [
   "std",
 ] }
 
-# Depending on the release cycle, this dependency might want an earlier version of the FVM.
-# We can work around it by hardcoding the method hashes; currently there is only one.
-# frc42_dispatch = "3.2"
+# We need a version of frc42_dispatch that has been updated to use FVM 4.1
+frc42_dispatch = { git = "https://github.com/fridrik01/filecoin.git", branch = "update-fvm-4.1" }
```

Comment:
Note, that I needed to fork this repo to update it to FVM 4.1. I have a PR for merging this into the official repo (https://github.com/helix-onchain/filecoin/pull/231)